src/Symfony/Component/Workflow/Workflow.php

```diff
@@ -235,6 +235,23 @@ public function getEnabledTransitions(object $subject)
         return $enabledTransitions;
     }
 
+    /**
+     * {@inheritdoc}
+     */
+    public function getEnabledTransition(object $subject, string $name)
```

Comment:
As a new method you can add return typehint i think here

Reply:
yes !
Thank you :)